app/controllers/common_pages_controller.rb

```diff
@@ -1,10 +1,39 @@
 # frozen_string_literal: true
 class CommonPagesController < ApplicationController
-  skip_before_action :authenticate_user!, :require_active_user, only: [:home]
+  helper StatisticsHelper
+  include Statistics
+  before_action :status_inspector, only: [:statistics]
+  skip_before_action :authenticate_user!, :require_active_user,
+                     only: [:home, :statistics]
 
   def home
     @issues_feed = Issue.approved.ordered.limit(4)
     @count = Issue.approved.count
     @closed_count = Issue.closed.count
   end
+
+  def statistics
+    @by_time_opened = Issue.statistics_for(period, group_by,
```

Comment:
this method is ok?
you have method: "xxx_opened" .. but 
%w(opened closed) - for opened you returning closed too?
and
%w(closed closed) <- looks strange